Makefile

```diff
@@ -8,13 +8,13 @@ ci: build
 
 # INSTALL ######################################################################
 
-VENDOR_DIR := vendor
+VENDOR_DIR := ~/.vendor/homepage
```

Comment:
You should not write files into the project dir when using Vagrant, mostly because of problems with VirtualBox file shares and Windows file locking issues. In this particular case, one of the gems refuses to install due to file locking issues. You can find more info at http://joequery.me/guides/bundle-install-text-file-busy/